core/src/main/java/tech/tablesaw/io/csv/CsvReader.java

```diff
@@ -146,74 +148,86 @@ private CsvReader() {
     }
 
     public static Table read(CsvReadOptions options) throws IOException {
-        ColumnType[] types = options.columnTypes();
-        byte[] bytes = options.reader() != null
+
+        final byte[] bytes = options.reader() != null
```

Comment:
adding `final` everywhere is inconsistent with the style in the rest of the codebase where we only use `final` for class members (or if needed because a variable is referenced from inside a closure). do you mind removing?